src/sparseml/exporters/transforms/convert_quantizable_matmul.py

```diff
@@ -1,161 +0,0 @@
-# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
```

Comment:
did you `git add` this file after deleting? looks like git didn't pick up the rename/deletion

Reply:
It's because theres too many differences - I moved the function to a method of a different name and renamed the class